sqlitecluster/SQLite.cpp

```diff
@@ -704,12 +704,17 @@ void SQLite::setUpdateNoopMode(bool enabled) {
     if (_noopUpdateMode == enabled) {
         return;
     }
-    if (enabled) {
-        SQuery(_db, "enabling noop-update mode", "PRAGMA noop_update=ON;");
-    } else {
-        SQuery(_db, "disabling noop-update mode", "PRAGMA noop_update=OFF;");
-    }
+
+    // Enable or disable this query.
+    string query = "PRAGMA noop_update="s + (enabled ? "ON" : "OFF") + ";";
```

Comment:
is this a rogue `s` after the equal sign? I don't see it defined anywhere.

Reply:
It makes it a string. http://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s